ui/src/components/retreat/home/PraiseWall.js

```diff
@@ -0,0 +1,280 @@
+import {
+  Button,
+  Container,
+  Flex,
+  Heading,
```

Comment:
clean up the unused imports so it doesnt create warnings

Reply:
cleaned up excess imports